tests/test_meshtastic_utils.py

```
@@ -5299,5 +5370,21 @@ def submit_side_effect(_func, *_args, **_kwargs):
             mock_scan.assert_not_called()
 
 
+def test_none_startup_drain_event_is_safe_noop(reset_meshtastic_globals):
```

Ruff (0.15.9)

[warning] Unused function argument: `reset_meshtastic_globals`

(ARG001)